core/src/components/modal/modal.tsx

```diff
@@ -689,6 +689,21 @@ export class Modal implements ComponentInterface, OverlayInterface {
     return this.currentBreakpoint;
   }
 
+  private onHandleClick = () => {
```

Comment:
The default handle is tiny, so clicking on it is super fiddly. You can drag the whole header, so maybe it would make sense to allow clicking that instead? 🤔 Failing that, maybe some padding or a wrapper element or something around the handle to expand the clickable space?

Reply:
I think padding around the clickable space could make a lot of sense here. I wouldn't advise we add clicking to the header, as it would be different behavior than native.